pandas/tests/msgpack/test_unpack.py

```diff
@@ -62,3 +63,14 @@ def _hook(self, code, data):
         assert unpacker.unpack() == {'a': 123}
         unpacker.feed(packb({'a': ExtType(2, b'321')}, encoding='utf-8'))
         assert unpacker.unpack() == {'a': ExtType(2, b'321')}
+
+    def test_unpack_categorical_index(self):
```

Comment:
Thanks, I tried to change the it accordingly. Do you get a notification if I push new commits? Or should I comment?